examples/demo_interface.pp

```diff
@@ -56,4 +56,10 @@
     duplex      => 'auto',
     speed       => '100m',
   }
+  #  Requires F3 or newer linecards
```

Comment:
Agreed for beaker, but for the demo manifest? 
Won't the agent run fail if the entire provider is unsupported?